llm/llama_flags_cuda.go

```diff
@@ -0,0 +1,13 @@
+//go:build cuda
+// +build cuda
+
+package llm
+
+/*
+#cgo CFLAGS: -O3 -std=c11 -fPIC -DNDEBUG -pthread -march=native -mtune=native
```

Comment:
Be careful with `-march` and `-mtune`, it might not be supported on all compilers.

Reply:
I removed the changes to the base flags from this PR, this was trying to get it to run faster on Windows. You're right that it should be a separate change that we can properly test.